dscblog/static/styles/header.css

```diff
@@ -46,12 +46,104 @@
   box-shadow: 3px 3px 0px #9e9e9e;
   background-color: rgba(31, 31, 31, 0.856);
 }
-.hd_opt:hover {
+.hd_opt:hover, .notification-wrapper.active #notification-bell{
   box-shadow: 5px 5px 0px #989898;
   background-color: rgb(255, 255, 255);
   color:rgb(46, 46, 46);
 }
-[data-theme="dark"] .hd_opt:hover {
+
+.notification-wrapper .notification{
+  width: 400px;
+  max-height: 250px;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 50px;
+  right: 10px;
+  overflow-y: scroll;
+  background: rgba(219, 219, 219, 1);
+  border: 1px solid rgba(0,0,0,.15);
+  box-shadow: 0 6px 12px rgba(0,0,0,.175);
+  z-index: 0;
+  opacity: 0;
+  transition: all .3s ease-in;
+  -ms-overflow-style: none; /* IE and Edge */
+  scrollbar-width: none; /* Firefox */
+
+}
+.notification-wrapper .notification::-webkit-scrollbar {
+  display: none;
+}
+[data-theme="dark"] .notification-wrapper .notification{
+  background: rgb(55, 55, 55);
+}
+.notification-wrapper.active .notification{
+  z-index: 3;
+  opacity: 1;
+}
+.notification-wrapper .notification .alerts-wrapper{
+  text-align: left;
+  width: 100%;
+  margin-top: 5px;
+}
+.notification-wrapper .notification .alerts-wrapper:last-child{
+  margin-top: 0;
+}
+.notification-wrapper .notification .alerts-wrapper .type {
+  font-weight: 700;
+  border-bottom: solid 1px;
+  width: 100%;
+  display: block;
+  padding-left: 5px;
+  padding-bottom: 5px;
+
+}
+.notification-wrapper .notification .alerts-wrapper .alert{
+  margin: 10px 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  font-size: .9rem;
```

Comment:
font-size: .85rem;
color: var(--grey); //For older alerts
color: rgb(186 186 186); //For new alerts (dark m)
color: rgb(59 59 59); //For new alerts (light m)